### PR TITLE
Add support for Jupyter notebook container

### PR DESCRIPTION
**Pull request by eric-anderson**
Adjust instructions and example so that it works on both localhost and the container.

### CODE REVIEW

**Comment by eric-anderson** [comment]
You can try it by downloading the .env and compose.yaml files and specifying VERSION=latest in the environment of your docker commands.

**Review by jonfritz — APPROVED** [review]
LGTM!

File: compose.yaml [hunk 1]
```diff
@@ -92,7 +92,25 @@ services:
       - crawl_data:/tmp/crawl_data
     command: /bin/sh -c 'echo size-before && du -s /tmp/* && rm -rf /tmp/opensearch_data/* /tmp/crawl_data/* && echo size-after && du -s /tmp/*'
     
+  jupyter:
+    profiles: ['jupyter']
+    image: arynai/jupyter:${JUPYTER_VERSION}
```
Comment (austintlee) [inline compose.yaml:97]: Isn't there a public version that we can point to?

Comment (eric-anderson) [reply compose.yaml:97]: I don't understand the question. That is public.
https://hub.docker.com/repository/docker/arynai/jupyter/general

**Review by austintlee — APPROVED** [review]

**Pull request merged**